### PR TITLE
#172: Added Integer and Decimal as a possible type for all result att…

### PR DESCRIPTION
**Pull request by brynrhodes**
…ributes in QDM 5.3.

### CODE REVIEW

**Comment by edeyoung** [comment]
Bryn,

I've requested a review from Chris Moesel.

Best,
Lizzie

File: Src/java/qdm/src/main/resources/gov/healthit/qdm/qdm-modelinfo-5.3.xml [hunk 1]
```diff
@@ -257,6 +269,7 @@
         <ns4:element name="ordinality" type="System.Code"/>
         <ns4:element name="incisionDatetime" type="System.DateTime"/>
         <ns4:element name="negationRationale" type="System.Code"/>
+        <ns4:element name="component" type="list&lt;QDM.Component&gt;"/>
```
Comment (cmoesel) [inline Src/java/qdm/src/main/resources/gov/healthit/qdm/qdm-modelinfo-5.3.xml]: What's this?  Did it leak in from another PR?

File: Src/java/qdm/src/main/resources/gov/healthit/qdm/qdm-modelinfo-5.3.xml [hunk 2]
```diff
@@ -410,13 +425,16 @@
...
+                <ns4:choice xsi:type="ns4:NamedTypeSpecifier" modelName="System" name="Integer"/>
+                <ns4:choice xsi:type="ns4:NamedTypeSpecifier" modelName="System" name="Decimal"/>
                 <ns4:choice xsi:type="ns4:NamedTypeSpecifier" modelName="System" name="Code"/>
                 <ns4:choice xsi:type="ns4:NamedTypeSpecifier" modelName="System" name="Quantity"/>
                 <ns4:choice xsi:type="ns4:NamedTypeSpecifier" modelName="QDM" name="Ratio"/>
             </ns4:typeSpecifier>
         </ns4:element>
         <ns4:element name="anatomicalLocationSite" type="System.Code"/>
         <ns4:element name="negationRationale" type="System.Code"/>
+        <ns4:element name="component" type="list&lt;QDM.Component&gt;"/>
```
Comment (cmoesel) [inline Src/java/qdm/src/main/resources/gov/healthit/qdm/qdm-modelinfo-5.3.xml]: What's this?  Did it leak in from another PR?

**Comment by cmoesel** [comment]
Also -- just to confirm, these changes shouldn't affect the XSD?

File: Src/java/qdm/src/main/resources/gov/healthit/qdm/qdm-modelinfo-5.3.xml (hunk 1 above)
Comment (brynrhodes) [inline Src/java/qdm/src/main/resources/gov/healthit/qdm/qdm-modelinfo-5.3.xml]: Yes, this is a change applied by the other QDM-related pull request, I must have inadvertently copied the source from that PR. Correcting.

File: Src/java/qdm/src/main/resources/gov/healthit/qdm/qdm-modelinfo-5.3.xml (hunk 2 above)
Comment (brynrhodes) [inline Src/java/qdm/src/main/resources/gov/healthit/qdm/qdm-modelinfo-5.3.xml]: Same issue, this is in the other PR.

**Comment by brynrhodes** [comment]
And yes, these changes don't affect the XSD, I'm not reflecting the choice types in the XSD, only the model info.

**Comment by brynrhodes** [comment]
Okay, submitted a fix for these issues. This PR will need to be merged first though, otherwise, the component changes from the other PR may be stomped.

**Pull request merged**